docs/fast_data/configuration/single_view_trigger_generator.md

```diff
@@ -87,3 +68,254 @@ The Kafka Projection Updates config map contains the `kafkaProjectionUpdates.jso
 
 Remember to copy/paste the mount path into the `KAFKA_PROJECTION_UPDATES_FOLDER` environment variable so the service can read the file.
 If you need more info on how to configure the `kafkaProjectionUpdates.json` file, please refer to the [Kafka Projection Updates](/fast_data/configuration/config_maps/kafka_projection_updates.md) page.
+
+### EVENT_STORE Config
+
+The `EVENT_STORE Config` is a JSON file containing the configuration of the consumer and producer of the service itself and it has the following format:
+
+```json
+{
+  "consumer": {
+    "kafka": {
+      // Kafka consumer configuration (see below)
+    }
+  },
+  "producer": {
+    "<kafka | mongo>": {
+      // Kafka or mongo producer configuration (see below)
+    },
+  }
+}
+```
+
+:::caution
+Mind that only one producer and consumer must be configured at a time so the service knows which kind to use. Providing more than one consumer or producer will fail the configmap validation and shut down the service at start up.
```

Comment:
do we have some specific error code / error handling that clearly states this problem in the logs of the service? do we need to add this to the error codes page related to SVTG?

Reply:
It will just throw a validation error of the config map with error code `FD_SVTG_E0007`. the message will be something like, "producer cannot have more than one property" so it should be pretty straight forward